nominations/models.py

```diff
@@ -91,9 +91,10 @@ class NominationsPlatformAlert(models.Model):
         help_text='Show alert on nominations platform pages.'
     )
 
-    alert_level = models.IntegerField(
-        choices=ALERT_LEVELS,
-        default=3,
+    alert_level = models.CharField(
+        max_length=16,
```

Comment:
Suggest to add code comment or used a named variable etc., otherwise this is a confusing "magic number" that seems to come from nowhere.

Reply:
👍